game-core/src/main/java/games/strategy/triplea/delegate/battle/MustFightBattle.java

```diff
@@ -1687,32 +1681,6 @@ private void checkForUnitsThatCanRollLeft(final IDelegateBridge bridge, final bo
     }
   }
 
-  private void standardAttackersFire() {
-    findTargetGroupsAndFire(
```

Comment:
Have you considered inlining these methods 'standardAttackersFire' and 'standardDefendersFire'? WDYT about that idea? Otherwise, can you clarify the motivation or benefit for having them?

As-is, from a casual perspective, we have a one-line method (the IExecutable) that uses none of the passed in args, which invokes a single use method that is itself a one-liner. Usually such patterns are candidates for inlining to reduce indirection. Perhaps I'm missing something (?)  (an answer to the above questions might point out what that is)

Reply:
I'm confused.  I did inline this in the OffensiveStandard and DefensiveStandard class.  It looks like you might have gotten the deleted lines and the added lines backwards.

Reply:
hah, true enough : ) 

Sorry about that.